api/KHR_parallel_shader_compile.json

```
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "KHR_parallel_shader_compile": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/KHR_parallel_shader_compile",
+        "support": {
+          "chrome": {
+            "version_added": "76"
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": "79"
+          },
+          "firefox": {
+            "version_added": "80"
+          },
+          "firefox_android": {
+            "version_added": null
```

Since Firefox for Android hasn't had a release in ages, it seems implausible that this would be anything other than `false`

```suggestion
            "version_added": false
```